cmd/data-usage-cache.go

```diff
@@ -469,7 +477,11 @@ func hashPath(data string) dataUsageHash {
 		data = strings.Trim(data, hashPathCutSet)
 	}
 	data = path.Clean(data)
-	return dataUsageHash(xxhash.Sum64String(data))
+	a, b := siphash.Hash128(0, 0, []byte(data))
```

Comment:
I would propose e.g. `blake2b` or a SHA variant here. SipHash is not a hash function.